devel/minipro/Portfile

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
 
-github.setup        vdudouyt minipro 0.0.1
+name                minipro
+version             0.5
 categories          devel
-maintainers         openmaintainer {gmail.com:ranauei @ranauei}
+maintainers         openmaintainer {krischik @krischik}
```

Comment:
You're right, GitHub handle @ranauei doesn't exist today. I feel fairly certain that it did exist when I added it to the ports this person maintains in #1796.

Did you contact this person by email to discover if they have a new GitHub handle or if they still wish to maintain this port?

This person maintains other ports. The maintainers line of all the ports should be fixed simultaneously.

Reply:
I've now filed a [port abandonment ticket](https://trac.macports.org/ticket/62205) for this.

Reply:
Here is why I didn't:

I first saw this was a ticket submission from 6 years ago, committed by Kurt. The submitted version was 0.0.1.

We have never seen or heard from the supposed maintainer since, in five years.

All the changes in the port since being committed initially were mostly done by you and larry.

The supposed maintainer has no other ports.

So -- IMHO -- this did not need a port abandonment ticket, as it was clearly either abandoned either at birth or immediately post-partum.

Reply:
As I said above:

> This person maintains other ports. The maintainers line of all the ports should be fixed simultaneously.

Reply:
that is fair.